docs/user/started.md

```diff
@@ -127,23 +127,23 @@ Now that we understand `Apps`, we can practice some data processing on your proj
 
 ![create process](/docs/img/empty.process.png)
 
-3. Nice! In order to process the data, you first need to stage data from your `Archive` to your process. Each process can only process data that is either staged or generated by other Apps. You can find the `Stage New Data` button on the buttom of the screen, click it.
+3. Nice! In order to process the data, you first need to stage data from your `Archive` to your process. Each process can only process data that is either staged or generated by other Apps. You can find the `Stage New Data` button on the bottom of the screen, click it.
 
-![stage data](/docs/img/select.process.png)
+![stage data](/docs/img/stage%20data.png)
```

Comment:
@guaje good catch. Thanks for finding the correct images. @soichih we might want to avoid blank spaces in file names so that we are more compatible with terminals (see "%20").